core/vm/intpool.go

```diff
@@ -65,3 +68,40 @@ func (p *intPool) put(is ...*big.Int) {
 		p.pool.push(i)
 	}
 }
+
+const poolDefaultCap = 25
+
+// intPoolPool manages a pool of intPools.
+type intPoolPool struct {
+	pools []*intPool
+
+	// Mutex
```

Comment:
I don't think you need this comment here, just remove it + the empty line. It's fairly obvious.